rum_react/README.md

```diff
@@ -2,65 +2,152 @@
 
 ## Overview
 
-With the Datadog [React integration][1], resolve performance issues quickly in React components by:
+With the Datadog RUM React integration, resolve performance issues quickly in React components by:
 
 - Debugging the root cause of performance bottlenecks, such as a slow server response time, render-blocking resource, or an error inside a component
-- Automatically correlating React performance data with user journeys, AJAX calls to the server side, and logs
-- Alerting your engineering teams when crucial performance metrics for React (such as Core Web Vitals) fall below a threshold that results in a poor user experience
-
+- Automatically correlating Web performance data with user journeys, HTTP calls, and logs
+- Alerting your engineering teams when crucial Web performance metrics (such as Core Web Vitals) fall below a threshold that results in a poor user experience
 
 Monitor your React applications from end-to-end by:
 
 - Tracking and visualizing user journeys across your entire stack
-- Debugging the root cause of slow load times, which may be an issue with your React code, network performance, or underlying infrastructure 
+- Debugging the root cause of slow load times, which may be an issue with your React code, network performance, or underlying infrastructure
 - Analyzing and contextualizing every user session with attributes such as user ID, email, name, and more
 - Unifying full-stack monitoring in one platform for frontend and backend development teams
 
 ## Setup
 
-### Collect RUM events 
+First, make sure to [setup Datadog RUM][1] in your application.
 
-To start collecting Real User Monitoring events from your React application, see [React Monitoring][2].
+Then, install the `@datadog/browser-rum-react` NPM package using your package manager of choice. For example:
```

Comment:
Should we point to the RUM Application creation page here instead of repeating installation steps? That would avoid having to maintain the instructions in both places.

Reply:
Interesting. I changed the setup section, let me know what you think!